server/repository/src/migrations/v1_06_00/clinician_store_join_add_clinician_link_id.rs

```diff
@@ -23,18 +23,16 @@ pub(crate) fn migrate(connection: &StorageConnection) -> anyhow::Result<()> {
             ADD COLUMN clinician_link_id TEXT NOT NULL REFERENCES clinician_link (id) DEFAULT 'temp_for_migration'; 
             UPDATE clinician_store_join SET clinician_link_id = clinician_id;
             PRAGMA foreign_keys = ON;
-
-            CREATE INDEX "index_clinician_store_join_clinician_link_id_fkey" ON "clinician_store_join" ("clinician_link_id");
-            "#,
+        "#,
     )?;
 
-    sql! {
+    sql!(
         connection,
         r#"
-        DROP INDEX IF EXISTS index_clinician_store_join_clinician_id_fkey;
-        ALTER TABLE clinician_store_join DROP COLUMN clinician_id;
+            ALTER TABLE clinician_store_join DROP COLUMN clinician_id;
+            CREATE INDEX "index_clinician_store_join_clinician_link_id_fkey" ON "clinician_store_join" ("clinician_link_id");
```

Comment:
What is our naming convention for the indices? There are some entries like
`index_clinician_store_join_clinician_link_id` and some like `index_clinician_store_join_clinician_link_id_fkey`?

@andreievg you added mix of it in `server/repository/src/migrations/v1_06_00/indexes.rs`. Was the `fkey` just a typo or was there a reason behind it? which nameing convention should we use for new indices?

Reply:
I've been wondering the same! Where did the convention come from is it just us or is the naming generated by like _db browser for SQlite.app_?